benchmark/src/bench.rs

```diff
@@ -56,12 +57,21 @@ impl BenchmarkService for Benchmark {
     fn streaming_from_client(
         &mut self,
         ctx: RpcContext,
-        _: RequestStream<SimpleRequest>,
+        mut stream: RequestStream<SimpleRequest>,
         sink: ClientStreamingSink<SimpleResponse>,
     ) {
-        let f = sink.fail(RpcStatus::new(RpcStatusCode::UNIMPLEMENTED, None));
+        let f = async move {
```

Comment:
Seems it's a duplex streaming? So it's better to send a response for every request?

Reply:
// Single-sided unbounded streaming from client to server
// The server returns the client payload as-is once the client does WritesDone
rpc StreamingFromClient(stream SimpleRequest) returns (SimpleResponse);